src/axom/sidre/tests/sidre_mfem_datacollection.cpp

```diff
@@ -698,6 +699,11 @@ static void testParallelMeshReload(mfem::Mesh& base_mesh,
   auto reader_group_data = getGroupData(*reader_pmesh);
   EXPECT_EQ(writer_group_data, reader_group_data);
   EXPECT_TRUE(sdc_reader.verifyMeshBlueprint());
+
+  // Make sure that orientation checks pass
+  // These can fail if the nodal GF is not set up correction for a periodic (and HO?) mesh
+  EXPECT_EQ(sdc_reader.GetMesh()->CheckElementOrientation(), 0);
```

Comment:
Can you please expand on this a bit?

I guess we're assuming that the test meshes are orientable but possibly periodic. 
It might be good to note that (?)